### PR TITLE
Developer app page changes

### PR DESCRIPTION
**Pull request by nathankota**
Changes to images, technology types, and descriptions on the developer apps page, and to some of the subsequent pages.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -902,7 +902,7 @@ <h5 class="text-center">Environmental Map (DEQ)</h5>
         <h5 class="text-center">SGID Parcel Viewer (UGRC)</h5>
         <img alt="SGID Parcel Viewer (UGRC)" class="lozad website-thumb"
           src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
-          data-src="{% link images/parcels_utah_gov.png %}" />
+          data-src="{% link images/applications/parcels_utah_gov.png %}" />
       </a>
       <p class="muted">Zoom in to view parcel boundaries and basic attributes as assembled statewide by UGRC, in
```
Comment (steveoh) [inline index.html:907]: it'd be nice to add the plss app in here to have an even grid. I think we removed the crash app once upon a time and never replaced it.

**Review by steveoh — COMMENTED** [review]
This is a nice refresh. Great job so far.

We should update the specific pages also right? These pages seem out of sync in regard to the photos and the descriptions.

- https://deploy-preview-2213--gis-utah-gov.netlify.app/developer/applications/plss/
- https://deploy-preview-2213--gis-utah-gov.netlify.app/developer/applications/parcels/
- https://deploy-preview-2213--gis-utah-gov.netlify.app/developer/applications/raster/

 A couple suggestions for the tech used column...

I wonder if we remove it and put it under the description or on the specific page. This is a bigger ask so don't worry about it if you aren't enjoying yourself.

**Tech name updates**

1. ArcGIS API for JavaScript -> ArcGIS Maps SDK for JavaScript
    this might only be relevant where are using the 4x.
1. ReactJS via ViteJS -> React
1. WebAPI -> UGRC API
1. UGRC Widgets -> UGRC Components
    widgets were a dojo thing. components are a react thing.
1. Firebase * -> Firebase
    we're using more than you've mentioned and I think firebase is probably enough?
1. Discover -> Discover base maps
1. ArcGIS Server -> maybe remove?
1. Tailwind -> probably drop, we didn't mention css frameworks before

**Comment by nathankota** [comment]
I like these ideas. I knew the subsequent pages needed a refresh too, but didn't want it to keep me from proceeding with the main page. I'll start in on those sooner than later.

Incorporating the tech column into the description and/or specific pages sounds good. I'm going to make the suggested tech name changes first; add one more app (roadkill), and maybe reorder some others to put some of the most recent ones at the top, like roadkill and plss, unless you have strong feelings about the existing order? 

> This is a nice refresh. Great job so far.
> 
> We should update the specific pages also right? These pages seem out of sync in regard to the photos and the descriptions.
> 
> * https://deploy-preview-2213--gis-utah-gov.netlify.app/developer/applications/plss/
> * https://deploy-preview-2213--gis-utah-gov.netlify.app/developer/applications/parcels/
> * https://deploy-preview-2213--gis-utah-gov.netlify.app/developer/applications/raster/
> 
> A couple suggestions for the tech used column...
> 
> I wonder if we remove it and put it under the description or on the specific page. This is a bigger ask so don't worry about it if you aren't enjoying yourself.
> 
> **Tech name updates**
> 
> 1. ArcGIS API for JavaScript -> ArcGIS Maps SDK for JavaScript
>    this might only be relevant where are using the 4x.
> 2. ReactJS via ViteJS -> React
> 3. WebAPI -> UGRC API
> 4. UGRC Widgets -> UGRC Components
>    widgets were a dojo thing. components are a react thing.
> 5. Firebase * -> Firebase
>    we're using more than you've mentioned and I think firebase is probably enough?
> 6. Discover -> Discover base maps
> 7. ArcGIS Server -> maybe remove?
> 8. Tailwind -> probably drop, we didn't mention css frameworks before

**Comment by nathankota** [comment]
What app(s) does this apply to?

> **Tech name updates**
> 
> 1. ArcGIS API for JavaScript -> ArcGIS Maps SDK for JavaScript
>    this might only be relevant where are using the 4x.

File: developer/applications/atlas/index.html [hunk 2]
```diff
@@ -25,17 +25,17 @@ <h3>Tools for visualizing Utah Roads, Imagery, and Widgets</h3>
...
-  also uses most of the <a href="{% link developer/widgets/index.md %}">UGRC javascript widgets</a>. The popup makes
+  Atlas contains the complete and most current <a href="{% link discover/index.html %}">Discover base maps</a>. Atlas
+  also uses most of the <a href="https://github.com/agrc/kitchen-sink">UGRC Components</a>. Clicking anywhere on the map makes
   http requests to the <a href="{% link developer/web-services/index.md %}">UGRC Web API</a> to reverse geocode, search
-  for the city, county, land ownership, elevation, and US National Grid of the point clicked. For UGRC developers, Atlas
-  is a code base to keep improving its' usage of dojo and the esri js api. Atlas is often cloned and used as an
-  application boilerplate for new projects as it's structure is optimal for the dojo build process.</p>
+  for the city, county, land ownership, elevation, and US National Grid of the point clicked; and display the results in the left panel. 
+  For UGRC developers, Atlas is a code base to keep improving its usage of dojo and the ArcGIS Maps SDK for JavaScript.
```
Comment (steveoh) [inline developer/applications/atlas/index.html:37]:
```suggestion
  For UGRC developers, Atlas is a code base to keep improving its usage of React and the ArcGIS Maps SDK for JavaScript. 
```

**Pull request closed without merging**